test/Microsoft.AspNetCore.Hosting.Tests/WebHostBuilderTests.cs

```diff
@@ -237,99 +237,7 @@ public void ConfigureServices_CanBeCalledMultipleTimes()
             Assert.NotNull(host.Services.GetRequiredService<ServiceA>());
             Assert.NotNull(host.Services.GetRequiredService<ServiceB>());
         }
-
-        [Fact]
-        public void CodeBasedSettingsCodeBasedOverride()
-        {
-            var hostBuilder = new WebHostBuilder()
```

Comment:
These were real config tests (not server tests). How about converting it to test a different field like CaptureStartupErrors?